### ISSUE TITLE
InfluxDB 0.9 compatibility

### ISSUE THREAD

**Issue by tonioo** [mvonthron/munin-influxdb#6]
Hi,

I'm wondering if this tool is compatible with InfluxDB 0.9+ ?

Thanks.

**Comment by schemacs** [comment]
No.  I have this branch to support 0.9.x, but use at your risk: https://github.com/schemacs/munin-influxdb/tree/v0.9 . I will create a pr when my code is ready.
